src/napari_spatialdata/_scatterwidgets.py

```diff
@@ -23,6 +29,81 @@
 ]
 
 
+class SelectFromCollection:
+    """
+    Select indices from a matplotlib collection using `LassoSelector`.
+
+    Selected indices are saved in the `ind` attribute. This tool fades out the
+    points that are not part of the selection (i.e., reduces their alpha
+    values). If your collection has alpha < 1, this tool will permanently
+    alter the alpha values.
+
+    Note that this tool selects collection objects based on their *origins*
+    (i.e., `offsets`).
+
+    Parameters
+    ----------
+    ax : `~matplotlib.axes.Axes`
```

Comment:
we don't have types in docstrings, please remove and check that docstrings of this PR are consistent with rest of repo.